src/netius/extra/smtp_r.py

```
@@ -158,14 +181,25 @@ def relay(self, connection, froms, tos, contents):
         # is sent to all the recipients (better auto close support), note
         # that multiple SMTP session may be created for the message so that
         # all the hosts associated with the recipients are notified
-        callback = lambda smtp_client: smtp_client.close()
+        callback = lambda smtp_client: self.on_relay_smtp(
+            smtp_client, connection, froms, tos, contents
+        )
```

Ruff (0.15.9)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `callback` as a `def`

(E731)